assets/preload.json

```diff
@@ -0,0 +1,19 @@
+{
+  "resources": [
+    {
+      "type": "font/woff2",
```

Comment:
Include `as` too.

Reply:
Can the `as` value be reliably derived from the `type`?

Reply:
In the event we start including other things might become tricky. Not sure we would, but isn't onerous to just set it here.